config/env/.env.example

```
@@ -18,34 +18,28 @@ OBSERVABILITY_IP=127.0.0.1
-
-# Advanced: Container-specific tokens (uncomment if using multiple bots)
-# BUNKBOT_TOKEN=your-bunkbot-specific-token-here
-# DJCOVA_TOKEN=your-djcova-specific-token-here
-# COVABOT_TOKEN=your-covabot-specific-token-here
-# BLUEBOT_TOKEN=your-bluebot-specific-token-here
-
-# Legacy (not recommended)
-# TOKEN=your-fallback-token-here
+BUNKBOT_TOKEN=your-bunkbot-token-here
+DJCOVA_TOKEN=your-djcova-token-here
+COVABOT_TOKEN=your-covabot-token-here
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The COVABOT_TOKEN key should go before the DJCOVA_TOKEN key

(UnorderedKey)